lms/lmsfrontend/src/component/education/programmingcontestFE/EditTestCase.js

```diff
@@ -183,96 +185,106 @@ export default function EditTestCase(props) {
-          <Grid item xs={3}>
-            <input
-              type="file"
-              //accept=".c, .cpp, .java, .py"
-              id="selected-upload-file"
-              onChange={onFileChange}
+      <HustContainerCard title={'Testcase'}>
+        <Grid container spacing={2}>
+          <Grid item xs={2}>
+            <TextField
+              autoFocus
+              required
+              id="point"
+              label="Point"
+              placeholder="Point"
+              value={point}
+              onChange={(event) => {
+                setPoint(event.target.value);
+              }}
             />
           </Grid>
-          <br></br>
-          Description
-          <TextField
-            fullWidth
-            style={{
-              marginTop: "10px",
-              marginBottom: "24px",
-            }}
-            multiline
-            maxRows={10}
-            value={description}
-            onChange={(event) => {
-              setDescription(event.target.value);
-            }}
-          ></TextField>
-          Solution Output
-          <TextField
-            fullWidth
-            style={{
-              marginTop: "10px",
-              marginBottom: "24px",
-            }}
-            multiline
-            maxRows={10}
-            value={result}
-            onChange={(event) => {
-              setResult(event.target.value);
-            }}
-          ></TextField>
-          <Grid item xs={2}>
-            <Button
-              color="primary"
-              variant="contained"
-              type="submit"
-              onChange={onInputChange}
-              width="100%"
+          <Grid item xs={3}>
+            <TextField
+              autoFocus
+              // required
+              select
+              id="Public TestCase"
+              label="Public TestCase"
+              placeholder="Public TestCase"
+              style={{width: '150px'}}
+              onChange={(event) => {
+                setIsPublic(event.target.value);
+              }}
+              value={isPublic}
             >
-              SUBMIT
-            </Button>
-            <h2> Status: {uploadMessage}</h2>
+              <MenuItem key={"Y"} value={"Y"}>
+                {"Y"}
+              </MenuItem>
+              <MenuItem key={"N"} value={"N"}>
+                {"N"}
+              </MenuItem>
+            </TextField>
+            <br/>
           </Grid>
-          {isProcessing ? <CircularProgress/> : ""}
         </Grid>
-      </form>
+        <form onSubmit={handleFormSubmit}>
+          <Grid container spacing={1} alignItems="flex-end" mt={1}>
+            <Grid item xs={12}>
+              <Button color="primary" variant="outlined" component="label">
+                <PublishIcon/> Upload testcase file
+                <input hidden type="file" id="selected-upload-file" onChange={onFileChange}/>
+              </Button>
+            </Grid>
+            <br></br>
+            <Grid item xs={3}>
```

Comment:
use same type of TextField as Point, Public field above

<Grid item xs={3}>
  <Typography variant="h6">Description</Typography>
</Grid>
<Grid item xs={12}>
  <TextField
     ...
  />
</Grid>

=>
<Grid item xs={12}>
  <TextField
     label = "Description"
      ...
   />
</Grid>